std/typecons.d

```diff
@@ -146,7 +146,12 @@ public:
     ~this()
     {
         debug(Unique) writeln("Unique destructor of ", (_p is null)? null: _p);
-        if (_p !is null) delete _p;
+        if (_p !is null)
+        {
+            import core.memory : GC;
+            destroy(_p);
+            GC.free(cast(void*) _p);
```

Comment:
I'm not entirely sure if the GC.free should be here. It seems consistent with the previous behavior, and my expectation would be that Unique should free the memory it is holding when it goes out of scope, but opAssign only calls `destroy` not `GC.free`.